# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1384489243 18000
#      Thu Nov 14 23:20:43 2013 -0500
# Node ID 87db35b69cfd6f692b52c8236ce6d5226c7739c6
# Parent  a404853d20734714c9299699ac4cdf8151b34d5a
* configure.ac (XTRA_CRUFT_SH_LDFLAGS): Remove "-def" for MSVC.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1276,17 +1276,17 @@ if test x"$enable_static" = x"yes"; then
   STATIC_LIBS=yes
 else
   STATIC_LIBS=no
 fi
 
 XTRA_CRUFT_SH_LDFLAGS=
 if test $have_msvc = yes; then
   FLIBS="$FLIBS -lkernel32"
-  XTRA_CRUFT_SH_LDFLAGS="-Wl,-def:cruft/cruft.def"
+  XTRA_CRUFT_SH_LDFLAGS="-Wl,cruft/cruft.def"
 fi
 AC_SUBST(XTRA_CRUFT_SH_LDFLAGS)
 
 ### Check for BLAS and LAPACK libraries:
 
 ## Need to adjust FFLAGS to include correct integer size.
 save_FFLAGS="$FFLAGS"
 FFLAGS="$FFLAGS $F77_INTEGER_8_FLAG"
